CHANGES.md

```diff
@@ -3,6 +3,7 @@
 Grammars:
 
 - enh(swift) add SE-0335 existential `any` keyword (#3515) [Bradley Mackey][]
+- enh(swift) add support for `distributed` keyword [Marcus Ortiz][]
 
 [Bradley Mackey]: https://github.com/bradleymackey
```

Comment:
You need to add a link ref for your name here as well, it doesn't matter if it's way down below somewhere, each release needs their own link references.

Reply:
Got it, thanks. [Updated](https://github.com/highlightjs/highlight.js/pull/3523/commits/69eb3f1ddbcdf85f55477db1542ee7a9f4340b5f)